### DISCUSSION TITLE
Netlify failing to load

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38405] [New to GitHub]
Hello, I'm getting an error when trying to connect GitHub to Netlify as the first step of deployment. What can I do to fix this issue?
![Screenshot 2022-11-06 210330](https://user-images.githubusercontent.com/76718219/200165175-5f9c96c1-1893-4fe0-8e3f-515e4282c87e.png)

**Comment by jfullstackdev** [comment]
I'm seeing a lot of these errors in the past few days and from a long time ago,

maybe it's not permanent just temporary

but you can always ask them, fortunately they are on GitHub too, so you can raise an issue asking them

https://github.com/netlify

**Comment by Thanyanit-J** [comment]
This may be what you are looking for:
https://answers.netlify.com/t/netlify-times-out-when-trying-to-import-a-project-from-github/76528
